dist/index.js

```diff
@@ -1330,6 +1330,10 @@ const cache = __importStar(__webpack_require__(365));
 function run() {
     return __awaiter(this, void 0, void 0, function* () {
         try {
+            core.info('------------------------');
+            core.info('NOTE: this action is deprecated and will not be maintained in future by GitHub engineers.');
+            core.info('Please, migrate to https://github.com/ruby/setup-ruby from official Ruby community.');
```

Comment:
The second sentence here looks odd to me because there is no "official Ruby community". I'm not sure it really matters who is maintaining or not maintaining anything in any event. This is directed at people already using this action and you're making an explicit recommendation to them about what to use instead.

I'd suggest something like:

> NOTE: This action is deprecated and is no longer maintained. 
Please migrate to the "ruby/setup-ruby" action, which is being actively maintained.

I also capitalized the "This" after "NOTE" to better match the style used in the diff of the README elsewhere in this PR.

If you want to keep the original text, please note that the second sentence is missing an article. It should be "from **the** official Ruby community."